grantnav/frontend/tests.py

```diff
@@ -53,23 +53,6 @@ def test_search(provenance_dataload, client):
     assert response.context['text_query'] == 'gardens AND fundingOrganization.id:GB-CHC-1156077'
     assert response.context['results']['hits']['total']['value'] == 7
 
-    # click facet
-    wolfson_facet = response.context['results']['aggregations']['fundingOrganization']['buckets'][0]
-    assert wolfson_facet['doc_count'] == 7
-    assert wolfson_facet['key'] == '["Wolfson Foundation", "GB-CHC-1156077"]'
-    json_query = json.loads(urllib.parse.parse_qs(initial_response.url.split('?')[-1])['json_query'][0])
-
-    json_query['query']['bool']['filter'][0]['bool']['should'] = [{'term': {'fundingOrganization.id_and_name': '["Wolfson Foundation", "GB-CHC-1156077"]'}}]
-    assert json.loads(urllib.parse.parse_qs(wolfson_facet['url'].split('?')[-1])['json_query'][0]) == json_query
-
```

Comment:
We may as well keep the other assert tests e.g. `doc_count`, `key` and only remove the `assert json.loads(...` part

Reply:
kept some more of the test